crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -56,51 +56,60 @@ impl TransactionSubmitting for CustomNodesApi {
         &self,
         tx: TransactionBuilder<DynTransport>,
     ) -> Result<TransactionHandle> {
-        tracing::info!("Custom nodes submit transaction entered");
+        tracing::debug!("Custom nodes submit transaction entered");
         let transaction_request = tx.build().now_or_never().unwrap().unwrap();
         let mut futures = self
             .nodes
             .iter()
-            .map(|node| {
-                async {
-                    tracing::info!("Sending transaction...");
-                    match transaction_request.clone() {
+            .enumerate()
+            .map(|(i, node)| {
+                let label = format!("custom_nodes_{i}");
+                let transaction_request = transaction_request.clone();
+                async move {
+                    tracing::debug!(%label, "sending transaction...");
+                    let result = match transaction_request {
                         Transaction::Request(tx) => node.eth().send_transaction(tx).await,
-                        Transaction::Raw { bytes, hash: _ } => {
+                        Transaction::Raw { bytes, .. } => {
                             node.eth().send_raw_transaction(bytes.0.into()).await
                         }
-                    }
+                    };
+
+                    (label, result)
                 }
                 .boxed()
             })
             .collect::<Vec<_>>();
 
         loop {
-            let (result, index, rest) = futures::future::select_all(futures).await;
-            let lable = format!("custom_nodes_{index}");
-            tracing::info!("Loop iteration with node: {}", lable);
+            let ((label, result), _, rest) = futures::future::select_all(futures).await;
+            tracing::debug!(%label, "custom node loop iteration");
```

Comment:
I think this log can also be dropped because every arm of the following `match` logs the label and even what the outcome is.